content/post/id2vec.md

```diff
@@ -39,89 +39,60 @@ For example, let's take photos of cats and dogs. Given just arrays of pixels, we
 tell what is present on the images, however, imagine that we are given a black box which outputs
 the following:
 
-<style>
-table code {
-  background: none !important;
-  white-space: nowrap;
-}
-</style>
-
-<div class="table-2"></div>
+{{% text size="large" %}}
+{{% table "fixedLayout" %}}
 
 | Photo                                                | Embedding |
 |:----------------------------------------------------:|:---------:|
-| ![one cat](/post/id2vec/cat_1_240.jpg)               |  `[1, 0]` |
-| ![one dog](/post/id2vec/dog_1_240.jpg)               |  `[0, 1]` |
-| ![five cats](/post/id2vec/cat_5_240.jpg)             |  `[5, 0]` |
-| ![four dogs](/post/id2vec/dog_4_240.jpg)             |  `[0, 4]` |
-| ![one cat and one dog](/post/id2vec/cat_dog_240.jpg) |  `[1, 1]` |
-
-
-<style>
-.table-2 + table th {
-  width: 50%;
-}
-
-.table-2 + table td {
-  font-size: 2em;
-}
-
-.table-blank + table {
-  margin: 1em 0;
-  border: none !important;
-}
-
-.table-blank + table thead, .table-blank + table th {
-  height: 0;
-  padding: 0 !important;
-}
-
-.table-blank + table thead, .table-blank + table th, .table-blank + table tr, .table-blank + table td {
-  background-color: rgba(0, 0, 0, 0) !important;
-  border: none !important;
-}
-</style>
-
-Apart from the easy classification, we are now able to easily estimate the semantic difference
-between the images:
-
-<style>
-.table-img-160 + table img {
-  width: 160px;
-  object-fit: contain;
-}
-.table-img-160 + table td {
-  font-size: 2em;
-}
-</style>
-
-<div class="table-blank table-img-160"></div>
+| ![one cat](/post/id2vec/cat_1_240.jpg)               |   [1, 0]  |
+| ![one dog](/post/id2vec/dog_1_240.jpg)               |   [0, 1]  |
+| ![five cats](/post/id2vec/cat_5_240.jpg)             |   [5, 0]  |
+| ![four dogs](/post/id2vec/dog_4_240.jpg)             |   [0, 4]  |
+| ![one cat and one dog](/post/id2vec/cat_dog_240.jpg) |   [1, 1]  |
+
+{{% /table %}}
+{{% /text %}}
+
+Apart from the easy classification, we are now able to easily estimate the semantic difference between the images:
+
+{{% text size="large" %}}
+{{% table "noCellHighlighting" "noHeader" %}}
 
 |                                          |     |                                        |     |                                        |
 |:----------------------------------------:|:---:|:--------------------------------------:|:---:|:--------------------------------------:|
-| ![one cat](/post/id2vec/cat_dog_240.jpg) | `-` | ![one dog](/post/id2vec/dog_1_240.jpg) | `=` | ![one cat](/post/id2vec/cat_1_240.jpg) |
-| `[1, 1]`                                 |     | `[0, 1]`                               |     | `[1, 0]`                               |
+| ![one cat](/post/id2vec/cat_dog_240.jpg) |  -  | ![one dog](/post/id2vec/dog_1_240.jpg) |  =  | ![one cat](/post/id2vec/cat_1_240.jpg) |
+| [1, 1]                                   |     | [0, 1]                                 |     | [1, 0]                               |
```

Comment:
It must be a monospace font which represents the code, do we really need to remove "`"?

Reply:
Displaying it as "code" will use the "code" style, what means "grey background" what would looks even worst.

But there is maybe a kind of workaround:
Since we already have a `text` snippet &mdash;that lets you you to configure the font size&mdash;, we could let the user to select between a list of fonts; that way it could be done, for example:
`{{% text font="monospaced" %}}={{% /text %}}`

Reply:
Yes, that works very well.